### PR TITLE
🔥✨ Adapt image property changes

### PR DESCRIPTION
**Pull request by kirrg001**
refs https://github.com/TryGhost/Ghost/issues/8348

- [x] test signup
- [x] test all image cases again

### CODE REVIEW

**Comment by kevinansfield** [comment]
@kirrg001 there shouldn't be any underscored names in the client, all properties/attributes/methods are camelCase so for example `feature_image` should be `featureImage`

**Comment by kirrg001** [comment]
@kevinansfield 👍 
[refs https://github.com/TryGhost/Ghost-Admin/blob/master/app/serializers/application.js#L20-L22]

**Comment by kirrg001** [comment]
WIP again, testing something with image deletion.

**Comment by kirrg001** [comment]
Ok this is ready to review now.

File: app/models/setting.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default Model.extend(ValidationEngine, {
     title: attr('string'),
     description: attr('string'),
     logo: attr('string'),
-    cover: attr('string'),
+    cover_image: attr('string'),
```
Comment (kevinansfield) [inline app/models/setting.js]: Is there a reason the API is inconsistent here? There's an odd mix of camelCase vs underscore_case...

```
ghost_head
ghost_foot
cover_image
```


```
defaultLang
activeTimezone
forceI18n
activeTheme
isPrivate
```

Comment (kirrg001) [inline app/models/setting.js]: Good question. There is no reason, it's just inconsistent. Server uses camel case for our model schema and settings is a model. We should actually make this consistent before releasing the beta.

Comment (kevinansfield) [inline app/models/setting.js]: 👍 shall we tackle that in a separate issue?

Comment (kirrg001) [inline app/models/setting.js]: I will try to make it consistent now, if i can't make it today i will raise an issue for later.

**Comment by kirrg001** [comment]
@kevinansfield Thanks!

I have pushed it as separate commit https://github.com/TryGhost/Ghost-Admin/pull/661/commits/d588f21aef5e6cc55aee42c911695527b0dbfdea

**Pull request merged**